src/common/routes.js

```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
Yes, but if the dashboard is also meant to be a public profile page (as suggested for future in UX docs) there would need to be 2 views of dashboard (edit and read only).

But either way - it doesn't affect us now.